packages/material-ui-lab/src/AvatarGroup/AvatarGroup.test.js

```diff
@@ -39,4 +39,16 @@ describe('<AvatarGroup />', () => {
     expect(container.querySelectorAll('img').length).to.equal(1);
     expect(container.textContent).to.equal('+2');
   });
+
+  it('should display all avatars instead of `max` avatars followed by "+1"', () => {
+    const { container } = render(
+      <AvatarGroup max={2}>
+        <Avatar src="broken-url" />
+        <Avatar src="broken-url" />
+        <Avatar src="broken-url" />
+      </AvatarGroup>,
+    );
+    expect(container.querySelectorAll('img').length).to.equal(3);
+    expect(container.textContent).to.equal('');
```

Comment:
I think that we should expect
```suggestion
    expect(container.textContent).to.equal('+2');
```

Reply:
Hum, actually, I'm expecting the `textContent` to be empty.
But note that `querySelectorAll('img').length` should be 3

Example:
3 `<Avatar />`s with a `max` of 2 should display all avatars (instead of 2 avatars + "+1").
```jsx
<AvatarGroup max={2}>
  <Avatar src="broken-url" />
  <Avatar src="broken-url" />
  <Avatar src="broken-url" />
</AvatarGroup>
```

But good point, `.to.equal('')` is not eloquent. I'm changing the `.to.equal('')` to `to.be.empty`

Reply:
I think that if max === 2, then we should see, at most, 2 items (one with an avatar, another one with the number of extra items). Isn't it a tradeoff Atlassian took in https://codesandbox.io/s/lucid-lehmann-b8wik?file=/src/App.js:0-171?

Reply:
Hum..

https://www.chaijs.com/api/bdd
```javascript
expect([1, 2, 3]).to.not.be.empty;     // Not recommended
expect([1, 2, 3]).to.have.lengthOf(3); // Recommended
```

So, changing tests to
```javascript
expect(container.textContent).to.have.lengthOf(0);
```

Reply:
Ok, I see. So `max` should include the "+X" item. Give me a minute

Reply:
Well, it's my point of interrogation. What's more intuitive? My reflection goes with this assumption, the primary use case for the max prop is to accommodate fox the space available in the UI, they aim to bound the maximum space the component can take. If they have 4 items available, then we might always fill this space, at most, when max={4}.

Reply:
> Well, it's my point of interrogation. What behavior would be more intuitive?

Yeah. I think max should include "+X" item.

```jsx
<AvatarGroup max={2}>
  <Avatar src="image-url" />
  <Avatar src="image-url" />
  <Avatar src="image-url" />
</AvatarGroup>
```

should output

![image](https://user-images.githubusercontent.com/13006409/80419145-dff65e00-88d8-11ea-806a-9da788a1e8c1.png)